lib/gitlab/client/users.rb

```diff
@@ -127,13 +127,18 @@ def session(email, password)
     #
     # @example
     #   Gitlab.ssh_keys
+    #   Gitlab.ssh_keys(2)
     #
     # @param  [Hash] options A customizable set of options.
     # @option options [Integer] :page The page number.
     # @option options [Integer] :per_page The number of results per page.
     # @return [Array<Gitlab::ObjectifiedHash>]
-    def ssh_keys(options={})
-      get("/user/keys", query: options)
+    def ssh_keys(user_id=nil, options={})
```

Comment:
I'm not fond of changing the method signature.  What if we look for `:user_id` as part of options and call the different route if that key exists instead?